src/vm.rs

```diff
@@ -226,11 +226,19 @@ impl<T: GuestStagePageTable, S> Vm<T, S> {
 impl<T: GuestStagePageTable> Vm<T, VmStateInitializing> {
     /// Create a new guest using the given initial page table and vCPU tracking table.
     pub fn new(vm_pages: VmPages<T, VmStateInitializing>, vcpus: VmCpus) -> Self {
+        let vm_id = vm_pages.page_owner_id().raw();
         Self {
             vcpus,
             vm_pages,
             guests: None,
-            attestation_mgr: AttestationSha384::default(),
+            attestation_mgr: AttestationSha384::new(
+                // Fake compound device identifier (DICE CDI)
+                // TODO Get the CDI from e.g. the TSM driver.
+                b"THISISARANDOMCDI",
+                vm_id,
+                const_oid::db::rfc5912::ID_SHA_384,
+            )
+            .expect("Failed to create attestation manager"),
```

Comment:
Is it possible for this to fail? If so, then we shouldn't be using `expect()`. If not, can you put a comment briefly explaining why it can't fail?

Reply:
Yes, it can fail is the provided CDI is bogus or not formatted properly. I'll have `Vm::new()` return a `Result`.

Reply:
Fixed.